packages/flutter/lib/src/widgets/app.dart

```diff
@@ -50,9 +76,15 @@ class WidgetsApp extends StatefulWidget {
   /// Creates a widget that wraps a number of widgets that are commonly
   /// required for an application.
   ///
-  /// The boolean arguments, [color], [navigatorObservers], and
-  /// [onGenerateRoute] must not be null.
-  const WidgetsApp({
+  /// The boolean arguments, `color`, `navigatorObservers`, and
```

Comment:
please use `[...]` to refer to properties or arguments with the same name as properties.